api/src/error.rs

```diff
@@ -44,6 +44,14 @@ pub enum ResponseError {
     /// Wrapper for Diesel's error type
     #[fail(display = "Database error: {}", 0)]
     DieselError(#[cause] diesel::result::Error),
+
+    /// Wrapper for openid's client error type
+    #[fail(display = "{}", 0)]
+    OpenIdClientError(#[cause] ClientError),
+
+    /// Wrapper for openid's error type
+    #[fail(display = "{}", 0)]
+    OpenIdError(#[cause] OpenIdError),
```

Comment:
Not sure if this is on your branch since I did it recently, but on master these error types are broken up by client vs server. Could you move these around in the enum to fit?

Reply:
How should they be broken up? I think client error means the openid client messed up/ the provider messed up, Same with `OpenIdError`. Tbh i cant tell the diff btwn them